tictactoe/script-runner/build.gradle

```diff
@@ -24,18 +62,53 @@ task vdmClasspathJar(type: Jar) {
     }
 }
 
+ext {
+    scenarioFile = project.findProperty("scenarioFile") ?: "script.scn"
+    oracleScenarioFile = project.findProperty("oracleScenarioFile") ?: "oracle_script.scn"
+}
+
 task runScript(type: JavaExec) {
     dependsOn vdmClasspathJar
-    classpath = configurations.runner
-    main 'com.anaplan.engineering.azuki.runner.ScenarioScriptRunnerKt'
-    jvmArgs '-Xmx4g'
+    classpath = sourceSets.main.runtimeClasspath
+    mainClass = 'com.anaplan.engineering.azuki.tictactoe.scriptrunner.TicTacToeScenarioScriptRunnerKt'
+    jvmArgs '-Xmx4g', "--add-opens=java.base/java.nio=ALL-UNNAMED"
+    args = [
+        "-d", file("build/run${System.currentTimeMillis()}").absolutePath,
+        "-v", file("build/verified").absolutePath,
+        "-u", file("build/unverified").absolutePath,
+        "-i", file("config/imports").absolutePath,
+        "-t", "VDM",
+        file(scenarioFile).absolutePath,
+    ]
+    def instanceJars = [
+        vdmClasspathJar.archiveFile.get().asFile.absolutePath,
+    ]
+    systemProperties = [
+        "com.anaplan.engineering.azuki.implementation.instance.jars": instanceJars.join(","),
+    ]
+}
```

Comment:
It might be that I should revert this back so that it uses the barebones scenario runner by default, and we only use the heavyweight scenario runner for oracles.